src/styles/_board.scss

```diff
@@ -2,19 +2,231 @@
 @import 'board/dice';
 
 .board {
-  background-image: url(/images/board.png);
-  background-size: 100% 100%;
-  background-color: black;
+  background-size: 100%;
   width: 1000px;
   height: 1000px;
   margin: auto;
   position: relative;
+
   .top-box {
     width: 1000px;
     height: 140px;
     float: left;
   }
 
+  .board-border {
+    position: absolute;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+    background-image: url("/images/board/Ramka nano.png");
+    z-index: 1000;
+    pointer-events: none;
+  }
+
+  #p0 {
+    background: url("/images/board/Start.png");
+     background-size: cover;
```

Comment:
"background: url" and "background-size" should have the same indentation (powinny zawierać te same wcięcia)